integration-tests/opentelemetry-devservices/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+pingpong/mp-rest/url=${test.url}
+quarkus.rest-client.collector-api.url=http://localhost:16686
```

Comment:
This is internal Jaeger UI API. Out of interest, would it be too difficult to use GRPC/Protobuf API that is stable? https://www.jaegertracing.io/docs/1.22/apis/#grpcprotobuf-stable If so, let's stick with this one...